### DISCUSSION TITLE
چطوری از آیپی های مرتضی باشسیز استفاده کنیم

### DISCUSSION THREAD

**Discussion by Obeid1375** [hiddify/Hiddify-Manager#209] [Q&A سوالات رایج]
من هرچی زدم جواب نگرفتم روش خاصی داره ؟
روی دامنه cdn شده باید قرار داد یا دایرکت ؟

**Comment by ghermez** [comment]
روی دامنه cdn باید بزنی
آی پی رو چک کنید که پینگ داشته باشه
والبته گاهی با وجود پینگ اون آی پی، امکان دانلود نمیده!! 
در کل باید آزمون و خطا برین

این سایت رو هم ببینین: http://ip.flares.cloud
